pkg/kubelet/cm/devicemanager/manager.go

```diff
@@ -113,6 +113,7 @@ type ManagerImpl struct {
 	containerRunningSet sets.Set[string]
 
 	// update channel for device health updates
+	// TODO this chan whether should having a buffer, avoid blocking genericDeviceUpdateCallback when syncLoopIteration is executing other logic
```

Comment:
this makes sense. I checked other update chan and they are all buffered, which should avoid blocking the `syncLoopIteration` method.

Reply:
@SergeyKanzhelev Is this necessary?

Reply:
yes, adding buffer will be great

Reply:
@lengrongfu  can you please add an issue for this on GitHub instead of inserting this comment. Other than this, the change looks good

Reply:
Thanks, i will create a new issue and delete this TODO.